### PR TITLE
Added `defaultStyles` and `dialogStyle` properties

### PR DESCRIPTION
**Pull request by thejase**
Essentially, this allows customization of styles in two ways:

* Further extending inline styles for the `dialog`.
* Disabling all inline styles in the component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,20 +155,34 @@ Type: `Boolean`
 
 If `true`, the modal will receive a `role` of `alertdialog`, instead of its default `dialog`.
 
+### defaultStyles
+
+Type: `Boolean`
```
Comment (davidtheclark) [inline README.md]: Let's show the default value.

File: README.md [hunk 2]
```diff
@@ -155,20 +155,34 @@ Type: `Boolean`
 
 If `true`, the modal will receive a `role` of `alertdialog`, instead of its default `dialog`.
 
+### defaultStyles
```
Comment (davidtheclark) [inline README.md]: How about `includeDefaultStyles`? I think `include` makes it clear that the value is a boolean.

File: README.md [hunk 3]
```diff
@@ -155,20 +155,34 @@ Type: `Boolean`
...
+*Note:* `underlayStyle` can still be set inline, but these will be the only styles applied.
+
 ### dialogClass
 
 Type: `String`
 
 Apply a class to the dialog in order to custom-style it.
 
-Be aware that this module does apply various inline styles to the dialog element in order position it.
+Be aware that this module does apply various inline styles to the dialog element in order position it.  To disable *all inline styles*, see `defaultStyles`.
```
Comment (davidtheclark) [inline README.md]: One space after period. Also, change first sentence to make it clear that this behavior is only *by default*.

File: README.md [hunk 4]
```diff
@@ -268,7 +282,7 @@ Type: `string`
 
 Apply a class to the underlay in order to custom-style it.
 
-This module does apply various inline styles, though, so be aware that overriding some styles might be difficult. If, for example, you want to change the underlay's color, you should probably use the `underlayColor` prop instead of a class.
+This module does apply various inline styles, though, so be aware that overriding some styles might be difficult. If, for example, you want to change the underlay's color, you should probably use the `underlayColor` prop instead of a class.  If you would rather control *all CSS*, see `defaultStyles`.
```
Comment (davidtheclark) [inline README.md]: Same feedback as above for the similar sentence.

**Comment by thejase** [comment]
Thanks so much for the fast review. I will get all these suggestions in!

**Comment by thejase** [comment]
Ready for review.  Thanks!

File: README.md [hunk 5]
```diff
@@ -155,20 +155,34 @@ Type: `Boolean`
 
 If `true`, the modal will receive a `role` of `alertdialog`, instead of its default `dialog`.
 
+### includeDefaultStyles
+
+Type: `Boolean`, Default: `true`
+
+By default, styles are applied inline to the  dialog and underlay portions of the component.  However, you can disable all inline styles by setting `includeDefaultStyles` to `false`.  If set, *you must specify all styles externally*, including positioning.  This is helpful if your project uses external CSS assets.
+
+*Note:* `underlayStyle` can still be set inline, but these will be the only styles applied.
```
Comment (davidtheclark) [inline README.md]: Can't you also set `dialogStyle` now?

**Comment by thejase** [comment]
Glad I could contribute!  Thanks for working with me.

**Comment by davidtheclark** [comment]
Thanks again, @thejase! Released in 2.6.0.

**Comment by davidtheclark** [comment]
(And sorry for the delay on getting this out: it slipped through the cracks of my todo lists!)

**Comment by thejase** [comment]
No apologies!  Thanks again! :)

**Pull request merged**